loader/loader.go

```diff
@@ -119,6 +120,11 @@ func (l *loader) GetMatchingReleasePlanAdmission(ctx context.Context, cli client
 		return releasePlanAdmission, toolkit.GetObject(designatedReleasePlanAdmissionName, releasePlan.Spec.Target, cli, ctx, releasePlanAdmission)
 	}
 
+	// This is a temporary hack to work around RELEASE-1150. It should improved when that ticket is complete.
```

Comment:
Is this temporary? Wouldn't this be the outcome of that ticket?

Reply:
When I wrote that, I thought we would come up with something more clever than an if statement. Removed the comment